packages/calcite-components/src/components/table/table.tsx

```diff
@@ -217,6 +228,10 @@ export class Table extends LitElement implements LoadableComponent {
 
   // #region Private Methods
 
+  private handleSlotChange = (): void => {
```

Comment:
this could be a method rather than an arrow function

Reply:
I need the arrow function here so `this` refers to the component instance instead of the element.

Reply:
Never mind. I **can** access the instance method.

Reply:
> I need the arrow function here so this refers to the component instance instead of the element.

if you refactor to use this.listenOn(), it would auto-bind to `this`, so should be good